examples/chat2-reliable/chat.go

```diff
@@ -26,9 +28,14 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-// Chat represents a subscription to a single PubSub topic. Messages
-// can be published to the topic with Chat.Publish, and received
-// messages are pushed to the Messages channel.
+const (
+	maxMessageHistory   = 100
+	bloomFilterSize     = 10000
+	bloomFilterFPRate   = 0.01
```

Comment:
Would be interesting to understand the behaviour of the underlying library here. Presumably these parameters affect the number of hash functions applied to each added item? I also assume that false positivity rates increase with the number of items added, so not sure how the underlying library keeps the FPRate to 0.01 when we keep adding items to the filter?

My intuition was that we'll have to completely recompute the bloom filter every `t` minutes to ensure that we keep a limited number of IDs in the filter while still covering enough history to provide a useful ACK.